### PR TITLE
Add technohaven

### PR DESCRIPTION
**Pull request by mmmim24**

### CODE REVIEW

**Comment by mmmim24** [comment]
further added Ixora and sjinnovation

**Comment by mmmim24** [comment]
RiseUp Labs added

**Comment by mmahmoodictbd** [comment]
Please check comments

**Comment by mmmim24** [comment]
Can anyone please review these changes please?

**Comment by mmahmoodictbd** [comment]
> Can anyone please review these changes?

Can you please check my comments? TLDR; Put addresses in a single line, see the images I added.


Screenshot 2024-03-11 at 10 08 18

Screenshot 2024-03-11 at 10 08 41

**Comment by mmmim24** [comment]
done

File: README.adoc [hunk 1]
```diff
@@ -782,6 +798,11 @@ https://www.linkedin.com/company/southtechgroup/[LinkedIn]
 |PHP, CakePHP, Laravel, RabbitMQ, .NET, C#, django, Java, J2EE, Spring, WordPress, MySQL, MongoDB, PostgreSQL, SQL Server, Oracle, SCSS, LESS, jQuery, VueJS, Android, Ionic
 |https://tappware.com/[Website]
 
+|Technohaven Company Limited
+|Seleena Bhaban, House #169 (6th Floor), Road #03, Mohakhali DOHS, Dhaka 1206
+|NodeJS,Angular,Rust,React,Android,iOS,Machine Learning,Blockchain
```
Comment (arafat-hasan) [inline README.adoc:803]: add spaces after commas

File: README.adoc [hunk 2]
```diff
@@ -726,6 +736,12 @@ https://www.linkedin.com/company/medina-tech/[LinkedIn]
 |Web and Mobile application, Virtual, Augmented and Mixed reality, Motion Graphics, Animation, Visual Effects
 |https://singularitybd.com/[Website]
 
+|SJ Innovation
+|House - 281/A (Level - 1),
+Road - 19/C, New DOHS, Mohakhali, Dhaka - 1206  
+|React,Objective C,Java,Kotlin,Flutter,Swift,Blockchain
```
Comment (arafat-hasan) [inline README.adoc:742]: add spaces after commas

File: README.adoc [hunk 3]
```diff
@@ -672,6 +677,11 @@ https://www.linkedin.com/company/medina-tech/[LinkedIn]
 |Desktop, Mobile, Web, System & Network, Cloud Computing, AWS, Azure & Open Source DevOps
 |http://www.relisource.com[Website]
 
+|RiseUp Labs
+|Floor-14, Tropical Alauddin Tower, Plot No-32/C, Road-2, Sector-3, Uttara, Dhaka-1230  
+|Flutter,Swift,Kotlin,Typescript,React,Java,DJango,NodeJS,SQL etc.
```
Comment (arafat-hasan) [inline README.adoc:682]: add spaces after commas

File: README.adoc [hunk 4]
```diff
@@ -432,6 +432,11 @@ https://github.com/ChumbokIT[Github]
 |PHP, Laravel, Java, Kotlin, Dart
 |http://itmedicus.com/[Website]
 
+|IXORA Solution
+|Jahangir Tower (7th Floor), M-5, Section-14, Mirpur, Dhaka-1206.
+|React,Vue,TypeSCript,NodeJS,Django,SpringBoot,Flutter,Swift,SQL,MongoDB etc.
```
Comment (arafat-hasan) [inline README.adoc:437]: add spaces after commas

**Review by arafat-hasan — CHANGES_REQUESTED** [review]

**Review by arafat-hasan — CHANGES_REQUESTED** [review]
@mmmim24,  Would you please resolve the merge conflict?

**Review by mmahmoodictbd — APPROVED** [review]

**Comment by mmahmoodictbd** [comment]
I approved the PR, please fix the conflict then we can merge it.

**Pull request closed without merging**